src/materials/shaders/pointcloud.vert

```diff
@@ -370,6 +388,78 @@ vec3 getCompositeColor() {
 	return c;
 }
 
+void doClipping() {
+	bool insideAny = false;
```

Comment:
The rest of the codebase is using spaces so we should keep the consistency here as well.